python/api/provider/views.py

```diff
@@ -96,11 +96,14 @@ def get(self, request, provider_id, upgrade_id):  # pylint: disable=arguments-di
 class DoProviderUpgrade(GenericAPIPermView):
     queryset = Upgrade.objects.all()
     serializer_class = api.serializers.DoUpgradeSerializer
+    check_upgrade_perm = check_perm
```

Comment:
I don't think we should introduce new name here, it doesn't have any additional semantic